src/releng_channel_status/releng_channel_status/flask.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import
+
+import releng_channel_status
+from releng_channel_status.view import ChannelStatusView
+
+
+app = releng_channel_status.create_app()
+app.add_url_rule('/',
+                 defaults={'rule_alias': app.config.get('DEFAULT_ALIAS'), 'product': None, 'channel': None},
+                 view_func=ChannelStatusView.as_view('channel_status_default'))
+
+app.add_url_rule('/<rule_alias>/status',
```

Comment:
Nit: As the app is called "channel status", I think `/status` is redundant. `/<rule_alias>` or `/<product>/<channel>` sounds explanatory enough, to me